frontend/controller/serviceworkers/sw-primary.js

```diff
@@ -38,25 +174,26 @@ const sendMessageToClient = async function (payload) {
     client.postMessage(payload)
   }
 }
+*/
 
 self.addEventListener('message', function (event) {
-  console.debug(`[sw] message from ${event.source.id}. Current store:`, store)
+  console.debug(`[sw] message from ${event.source.id}.`)
```

Comment:
Should this debug message also include `event.data?.type`?

Reply:
Sure, although this log entry doesn't seem particularly useful.